aksel.nav.no/website/sanity/util.ts

```diff
@@ -1,12 +1,11 @@
-import {
-  ConditionalPropertyCallbackContext,
-  NewDocumentOptionsResolver,
-} from "sanity";
+import { ConditionalProperty, NewDocumentOptionsResolver } from "sanity";
 
-export const devsOnly = ({ currentUser }: ConditionalPropertyCallbackContext) =>
-  !currentUser?.roles.find(({ name }) =>
-    ["developer", "administrator"].includes(name),
-  );
+export function showForDevsOnly(): ConditionalProperty {
+  return ({ currentUser }) =>
+    !currentUser?.roles.find(({ name }) =>
+      ["developer", "administrator"].includes(name),
+    );
+}
```

Comment:
Why did you make this change?

Reply:
Feels its "cleaner" to use the top-level type for "hidden", rather than drilling down 3 levels
```ts
/** @public */
export declare type ConditionalProperty = boolean | ConditionalPropertyCallback | undefined

/** @public */
export declare type ConditionalPropertyCallback = (
  context: ConditionalPropertyCallbackContext,
) => boolean

/** @public */
export declare interface ConditionalPropertyCallbackContext {
  document: SanityDocument | undefined
  parent: any
  value: any
  currentUser: Omit<CurrentUser, 'role'> | null
}
```